package.json

```diff
@@ -1,17 +1,22 @@
 {
-  "name": "fireloop-starter",
+  "name": "fire-starter",
   "version": "1.0.0",
-  "description": "",
-  "main": "index.js",
+  "description": "Starter kit for the fireloop.io platform",
   "scripts": {
-    "preinstall": "npm install -g @angular/cli",
-    "install": "cd fireloop/ && npm install && cd ../webapp && npm install && npm install @angular/cli && cd ..",
+    "preinstall": "npm install -g @angular/cli @angular/compiler-cli",
```

Comment:
This will install `@angular/compiler-cli` globally for everyone using the starter. I don't think that's what we want, it's basically not needed, only on Heroku.

I don't think we should do any global installs on behalf of the user. The user might have a valid reason to keep a specific version of @angular/cli installed globally. 

Why not prefix all heroku specific scripts with `heroku-`?